Tests/Highcharts/ColorsTest.php

```diff
@@ -3,11 +3,12 @@
 namespace Ob\HighchartsBundle\Tests\Highcharts;
 
 use Ob\HighchartsBundle\Highcharts\Highchart;
+use PHPUnit\Framework\TestCase;
```

Comment:
Any idea why this [failed on Travis](https://travis-ci.org/marcaube/ObHighchartsBundle/jobs/303938429) with the following message using PHP 5.5 ?

> PHP Fatal error:  Class 'PHPUnit\Framework\TestCase' not found in /home/travis/build/marcaube/ObHighchartsBundle/Tests/Highcharts/ColorsTest.php on line 12